src/MemoryKeyboard/controller.js

```diff
@@ -0,0 +1,71 @@
+import { EventDispatcher } from '../eventDispatcher.js';
+import viewTemplate from './view.hbs';
+import stringToDom from 'string-to-dom';
+
+export const MemoryButtons = {
+    PLUS: 'plus',
+    MINUS: 'minus',
+    RECALL: 'recall',
+    CLEAR: 'clear'
+};
+
+export class MemoryKeyboard extends EventDispatcher {
+
+    /**
+     *  Constructor.  
+     */
+    constructor() {
+        super();
+        let viewString = viewTemplate();
+        this.view = stringToDom(viewString);
+        this._num = 0;
+        this.memoryPlusBtnClick();
```

Comment:
The method name is really bad, I though it was a handler, while it was actually binding a handler. Do similar approach like for operations, create the view here and call method `_bindEvents` (or similar).